bin/cron/deploy_to_test

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env ruby
+
+require 'cdo/chat_client'
+require 'cdo/github'
+require 'cdo/only_one'
+
+# @return [Boolean] Whether  the topic of Slack#developers includes "DTT: yes".
+def dtt_yes?
+  current_topic = Slack.get_topic('developers')
+  current_topic.include? 'DTT: yes'
+end
+
+# Updates the DTT portion of topic of Slack#developers to indicate robo-DTT.
+def update_developers_topic_robo_dtt
+  current_topic = Slack.get_topic('developers')
+  raise unless current_topic.include? 'DTT: yes'
+  new_topic = current_topic.gsub('DTT: yes', 'DTT: no (robo-DTT in progress)')
+  Slack.update_topic(new_topic)
+end
+
+# Updates the DTT portion of topic of Slack#developers to indicate DTT yes.
+def update_developers_topic_robo_failed
+  current_topic = Slack.get_topic('developers')
+  raise unless current_topic.include? 'DTT: no (robo-DTT in progress)'
+  new_topic = current_topic.gsub('DTT: no (robo-DTT in progress)', 'DTT: robo-DTT failed')
+  Slack.update_topic(new_topic)
+end
+
+def main
+  unless dtt_yes?
+    ChatClient.message(
+      'infra-test',
+      "robo-DTT skipped (\#developers topic is #{current_topic}).",
+      color: 'yellow'
+    )
+  end
+  update_developers_topic_robo_dtt
+
+  pr_number = GitHub.create_pull_request(
+    base: 'test',
+    head: 'staging',
+    title: 'DTT (Staging > Test) [robo-dtt]'
+  )
+  if pr_number.nil?
+    ChatClient.message(
+      'infra-test',
+      'GitHub.create_pull request failed.',
+      color: 'red'
+    )
+    update_developers_topic_robo_failed
+    return
```

Comment:
Ah, yes, fixed in upcoming PR. Thanks!

Note that I didn't want this script to raise an exception that wasn't rescued, that was my concern in my earlier comment.